.github/workflows/ci-docs.yml

```diff
@@ -0,0 +1,20 @@
+---
+# This workflow sets the "ci" status check to success in case it's a docs only PR and ci.yml is not triggered
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+name: ci # The name must be the same as in ci.yml
+
+on:
+  pull_request:
+    paths-ignore: # This expression needs to match the paths ignored on test.yml.
+      - '**'
+      - '!**/*.md'
+      - '!**/*.asciidoc'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
```

Comment:
I'm worried, this won't work as expected.

The normal job will create commit statuses with the matrix item name. `test / something`

This will just creat the `test` context.

Reply:
That's an interesting finding :/. 
Knowing the nature of this particular GitHub repository, I'll delete this workaround and the  paths-ignore